docs/kyma/docs/04-03-cluster-installation.md

```diff
@@ -0,0 +1,518 @@
+
+## Deploy Kyma
+
+1. Deploy Kyma using the `my-kyma` custom configuration file you created. Run:
+    ```
+    kubectl apply -f my-kyma.yaml
+    ```
+    >**NOTE:** If you get the `Error from server (MethodNotAllowed)` error, run the command again before proceeding to the next step.
+
+2. Check if the Pods of Tiller and the Kyma Installer are running:
+    ```
+    kubectl get pods --all-namespaces
+    ```
+
+3. Start Kyma installation:
+    ```
+    kubectl label installation/kyma-installation action=install
+    ```
+
+4. To watch the installation progress, run:
+    ```
+    while true; do \
+      kubectl -n default get installation/kyma-installation -o jsonpath="{'Status: '}{.status.state}{', description: '}{.status.description}"; echo; \
+      sleep 5; \
+    done
+    ```
+    After the installation process is finished, the `Status: Installed, description: Kyma installed` message appears.
+    In case of an error, you can fetch the logs from the Installer by running:
+    ```
+    kubectl -n kyma-installer logs -l 'name=kyma-installer'
+    ```
+
+## Add the xip.io self-signed certificate to your OS trusted certificates (optional)
+
+>**NOTE:** Skip this section if you use your own domain.
+
+After the installation, add the custom Kyma [`xip.io`](http://xip.io/) self-signed certificate to the trusted certificates of your OS. For MacOS, run:
+```
+tmpfile=$(mktemp /tmp/temp-cert.XXXXXX) \
+&& kubectl get configmap cluster-certificate-overrides -n kyma-installer -o jsonpath='{.data.global\.tlsCrt}' | base64 --decode > $tmpfile \
+&& sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain $tmpfile \
+&& rm $tmpfile
+```
+
+## Configure DNS for the cluster load balancer
+
+>**NOTE:** Execute instructions from this section only if you want to use your own domain.
+
+Run these commands:
+
+```
+export EXTERNAL_PUBLIC_IP=$(kubectl get service -n istio-system istio-ingressgateway -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+export REMOTE_ENV_IP=$(kubectl get service -n kyma-system application-connector-ingress-nginx-ingress-controller -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+export APISERVER_PUBLIC_IP=$(kubectl get service -n kyma-system apiserver-proxy-ssl -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+
+az network dns record-set a create -g $RS_GROUP -z $DNS_DOMAIN -n \*.$SUB_DOMAIN --ttl 60
+az network dns record-set a add-record -g $RS_GROUP -z $DNS_DOMAIN -n \*.$SUB_DOMAIN -a $EXTERNAL_PUBLIC_IP
+
+az network dns record-set a create -g $RS_GROUP -z $DNS_DOMAIN -n gateway.$SUB_DOMAIN --ttl 60
+az network dns record-set a add-record -g $RS_GROUP -z $DNS_DOMAIN -n gateway.$SUB_DOMAIN -a $REMOTE_ENV_IP
+
+az network dns record-set a create -g $RS_GROUP -z $DNS_DOMAIN -n apiserver.$SUB_DOMAIN --ttl 60
+az network dns record-set a add-record -g $RS_GROUP -z $DNS_DOMAIN -n apiserver.$SUB_DOMAIN -a $APISERVER_PUBLIC_IP
+```
+  </details>
+</div>
+
+### Access the cluster
+
```

Comment:
When you install with XIP you don't really know what the domain of the cluster is. 
To find out what the domain is, run: 
```
kubectl get virtualservice core-console -n kyma-system
```
This gets you the VS name of the cluster console which is basically the console address, such as `console.kyma.local`.